snuba/datasets/__init__.py

```diff
@@ -59,12 +59,12 @@ def default_conditions(self):
         """
         return []
 
-    def column_expr(self, column_name, body):
+    def column_expr(self, column_name, body, table_alias: str=""):
```

Comment:
I spent a very long time deciding if the default value should be "" or None. Finally in both cases the alias does not exists, avoiding a None input allows us to be more strict in type checking

Reply:
> avoiding a None input allows us to be more strict in type checking

I don't understand the reasoning here. Can you elaborate?

Between the two, `None` makes more sense to me, but I think I'm likely missing something.

Reply:
To me, it seems like it'd be clearer for `None` to be the default and an empty string just seems like an invalid alias. I don't have a strong opinion about it either, and don't think it's necessarily worth changing unless it's likely to cause a problem with accidental usage (probably not, but I'm not as familiar with this as you are.)